### ISSUE TITLE
About Accuracy

### ISSUE THREAD

**Issue by Ien001** [mengrang/Fine-grained-Recognition-Model-Compression#1]
hi, 
    I`m working on reimplementing  "Learning a Discriminative Filter Bank Within a CNN for Fine-Grained Recognition" now, but I cannot get the result in the paper, do you get the same accuracy listed in the paper?

Ian

**Comment by mengrang** [comment]
Do you implement the layers nonrandomly initiallation?it makes the key role in the accuracy

**Comment by Ien001** [comment]
Yeah.. but it seems like I'm failed on it, the accuracy is no different after I finish nonrandomly initiallation. 
Many ppl said that there are some tricks in the paper😳, I don't know, maybe I did sth wrong.

**Comment by mengrang** [comment]
Thanks for your comments, I am also intersted in the initialltion works.You can click [here](https://zhuanlan.zhihu.com/p/50083578) for My understanding of the initiallztion.

**Comment by fxle** [comment]
@Ien001 How do you try to nonrandomly initiallation?Can you share you idea? Thank you

**Comment by fxle** [comment]
@mengrang How accurate are you getting? In addition，Where to get these two files?'AgriculturalDisease_train_annotations.json'',"AgriculturalDisease_validation_annotations.json"

**Comment by mengrang** [comment]
The acc on my dataset is just OK, the two json files is my dataset

**Comment by Ien001** [comment]
Hi, I have read it, @mengrang 
it seems okay,
have u implement ur idea?

**Comment by doublemanyu** [comment]
Have you implement the non-random initialization? @mengrang @Ien001 @fxle

**Comment by chaerlo** [comment]
@Ien001 what‘s the accuracy u implement the non-random initialization？
